### DISCUSSION TITLE
不兼容的主题建议下架

### DISCUSSION THREAD

**Discussion by fxln0** [halo-dev/halo#5907] [General]
### 你当前使用的版本

2.15.2

### 描述一下此特性

halo长时间不更新和版本不兼容的主题建议下架了

**Comment by ruibaby** [comment]
感谢反馈，一般情况下，如果主题和插件不是造成了非常严重的错误，我们是不会主动下架应用的，并且考虑到以后三方开发者也会发布应用。

此外，这个问题不属于 issue，我将转移到 Discussion。
